influxdb3/management_cloud_decicated.go

```diff
@@ -0,0 +1,147 @@
+/*
+ The MIT License
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in
+ all copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ THE SOFTWARE.
+*/
+
+package influxdb3
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+type (
+	// CloudDedicatedClient represents a client for InfluxDB Cloud Dedicated administration operations.
+	CloudDedicatedClient struct {
+		client *Client
+	}
+
+	CloudDedicatedClientConfig struct {
+		AccountID        string
+		ClusterID        string
+		ManagementToken  string
+		ManagementAPIURL *url.URL
```

Comment:
Please add doc string what is used if the `ManagementAPIURL` is not specified.

Reply:
Addressed in [3abbb46](https://github.com/InfluxCommunity/influxdb3-go/pull/87/commits/3abbb46a86a5f2524087a5ec2e6ddf639faafcc2)